lib/jmap/mdn/send/mdn_send_method.dart

```diff
@@ -1,44 +1,36 @@
 import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
 import 'package:jmap_dart_client/http/converter/identities/identity_id_converter.dart';
-import 'package:jmap_dart_client/http/converter/mdn/mdn_converter.dart';
+import 'package:jmap_dart_client/http/converter/send/send_method_properties_converter.dart';
 import 'package:jmap_dart_client/http/converter/set/set_method_properties_converter.dart';
-import 'package:jmap_dart_client/jmap/account_id.dart';
 import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
-import 'package:jmap_dart_client/jmap/core/id.dart';
-import 'package:jmap_dart_client/jmap/core/method/method.dart';
-import 'package:jmap_dart_client/jmap/core/patch_object.dart';
+import 'package:jmap_dart_client/jmap/core/method/request/send_method.dart';
 import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
 import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:jmap_dart_client/jmap/mail/email/submission/set/set_email_submission_method.dart';
 import 'package:jmap_dart_client/jmap/mdn/mdn.dart';
 
-class SendMessageMethod extends MethodRequiringAccountId {
```

Comment:
What are EmailSubmissionId and ReferenceId field, can you clarify for me, please ?